src/App.tsx

```diff
@@ -54,6 +49,13 @@ import { ExportScriptFlyout } from "./components/ExportScriptFlyout";
 import { useRecordingContext } from "./hooks/useRecordingContext";
 import { StartOverWarningModal } from "./components/StartOverWarningModal";
 
+const cache = createCache({
```

Comment:
can you add some comments on how its used and whats its for?

Reply:
Basically there are issues with EUI that necessitate this. I've added a comment and a link to the relevant documentation. 0d71c584cfc307996d086f1e261fdaa818368723

Reply:
Thanks they helps